client/src/components/Carousel.jsx

```diff
@@ -1,8 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+
+//import share icon
+
+//import save icon
+import SaveIcon from './Saveicon.jsx';
+
+
+//create styled component for photo
+const StyledPhoto = styled.img`
+  width: 25%;
+  padding: 4px;
+`;
+
 const Carousel = (props) => {
+  {/* iterate over the props passed in and create a photo for each index */}
+  {for (var i = 0; i < props.photos.length; i++) {
+    //create a styled photo for each one
+    <StyledPhoto url={props.photos[i].url} />
+  }}
```

Comment:
is this code used anywhere? looks like the StyledPhoto is created but not stored in an array for later use

Reply:
You're right, I have to remove this!